aip/general/0127.md

```diff
@@ -78,8 +78,10 @@ message CreateBookRequest {
   - URIs **may** use nested fields for their variable names. (Additionally,
     AIP-134 mandates this for `Update` requests.)
   - URIs **must** use the `*` character to represent ID components, which
-    matches all URI-safe characters except for `/`. URIs **may** use `**` as
-    the final segment of a URI if matching `/` is required.
+    matches all URI-safe characters except for `/`.
+  - URIs **must not** use `**`, except in the final segment of the URI.
+  - URIs **should not** use `**` as the final segment of a URI unless doing so
+    for backwards-compatibility reasons ([AIP-122][]).
```

Comment:
I find this to be confusing. The first bullet makes it seem that `**` is allowed for the final segment, and the second that `**` **should not** be used as the final segment at all (unless for backwards compatibility). This seems contradictory.

Reply:
they're saying two different things:

1. is saying you can't use it anywhere except the last segment
2. is saying you shouldn't use it for the last segment either.

Although I think there's an easier way to say this: it should not be used, and if it is used it must only be used for the last segment .WDYT?